src/org/omegat/filters2/html2/HTMLOptions.java

```diff
@@ -106,22 +106,22 @@ public HTMLOptions(Map<String, String> options) {
      * Returns whether and when (X)HTML filter adds/rewrites encoding
      * declaration.
      *
-     * @return One of {@link REWRITE_MODE#ALWAYS}, {@link REWRITE_MODE#IFHEADER}
-     *         , {@link REWRITE_MODE#IFMETA}, {@link REWRITE_MODE#NEVER}.
+     * @return One of {@link RewriteMode#ALWAYS}, {@link RewriteMode#IFHEADER} ,
+     *         {@link RewriteMode#IFMETA}, {@link RewriteMode#NEVER}.
      */
-    public REWRITE_MODE getRewriteEncoding() {
-        return getEnum(REWRITE_MODE.class, OPTION_REWRITE_ENCODING, REWRITE_MODE.IFHEADER);
+    public RewriteMode getRewriteEncoding() {
```

Comment:
```suggestion
    RewriteMode getRewriteEncoding() {
```

This is better to be package-private.